app/models/portfolio_item.rb

```diff
@@ -32,7 +32,7 @@ def metadata
 
   def validate_deletable
     unless deletable?
-      errors.add(:base, "cannot be deleted because it is used by some order processes")
+      errors.add(:base, "cannot be deleted because it is used by order processes #{order_process_ids.uniq}")
```

Comment:
We need to call `order_process_ids` twice which needs to make DB queries.
You can delete `deletable?` method and combine the logic here to avoid redundant calls.